base/system/smss/pagefile.c

```diff
@@ -837,8 +837,8 @@ SmpCreateVolumeDescriptors(VOID)
     /* Query the device map so we can get the drive letters */
     Status = NtQueryInformationProcess(NtCurrentProcess(),
                                        ProcessDeviceMap,
-                                       &ProcessInformation,
-                                       sizeof(ProcessInformation),
+                                       &ProcessInformation.Query,
```

Comment:
This change looks fine to me though I'm curious what problem the other version caused.

Reply:
Thanks. Didn't see it stand out amount the other 53 commits ;)